quickstart/src/index.js

```diff
@@ -76,6 +76,15 @@ $.getJSON('/token', function(data) {
   };
 });
 
+// Get the Participant's Tracks.
+function getTracks(participant) {
+  return Array.from(participant.tracks.values()).filter(function(publication) {
+    return publication.track;
+  }).map(function(publication) {
+    return publication.track;
+  });
+}
```

Comment:
Should we import this from some common `utils.js` module?

Reply:
@markandrus We can move this to utils when we have more examples.